3.6/troubleshooting-cluster-agency-dump.md

```diff
@@ -8,50 +8,36 @@ How to produce an Agency Dump
 One can read out all information of an _Agency_ in the following way:
 
 ```
-curl -L http://<server>:<port>/_api/agency/read -d '[["/"]]'
+curl -k https://<any-coordinator>:<port>/_api/cluster/agency-dump > agency.json
 ```
 
-Please make sure to use the _IP_ (or hostname) and _PORT_ of one _Agent_.
+In case of an authenticated _Cluster_, to access _Agents_ a JWT token is needed.
```

Comment:
@kvahed Why do you need a JWT if talking to the Coordinator? Why mention Agents?